### ISSUE TITLE
Get Tic Tac Toe working

### ISSUE THREAD

**Issue by tengyifei** [TakefiveInteractive/TedkOS#71]
- [ ] Message queue
- [ ] Button widget
- [ ] Image widget

**Comment by C4Phone** [comment]
Sorry I clicked the wrong button... And btw could we just use Deque for message queue?

**Comment by tengyifei** [comment]
I think so.

**Comment by C4Phone** [comment]
Mouse is drawn under new window:

<img width="459" alt="screen shot 2015-12-15 at 10 29 10 am" src="https://cloud.githubusercontent.com/assets/7296488/11816602/cf592390-a316-11e5-900f-1029f71591aa.png">

**Comment by tengyifei** [comment]
Yeah I knew that. So we never showed it during demo. And it's laggy as sh*t
